docs/source/quick_start.rst

```diff
@@ -0,0 +1,193 @@
+============
+Quick Start
+============
+
+  This guide is intended to get new Kokkos users started in less than one hour.  Kokkos Core is the foundation for all other Kokkos Ecosystem projects.
+
+------------
+Requirements
+------------
+
+::
+
+  CMake >=3.18
+  gcc >= 8.3.1
+  CUDA >= 11
+  rocm >=5.7
+
+--------------------------------------
+Download Current Release, Set Up Build 
+--------------------------------------
+
+..
+  Choice 1 for obtaining current release:
+..
+
+* `Click to Download Current Release <https://github.com/kokkos/kokkos/archive/refs/heads/master.zip>`_ 
+
+
+..
+  Temp. Notes --
+  Buttons, badges, icons:
+  https://sphinx-design.readthedocs.io/en/latest/badges_buttons.html
+  :bdg-link-primary-line:`explicit title <https://example.com>`
+
+  https://getbootstrap.com/docs/5.0/components/badge/
+..
+
+
+..
+  Choice 2 for obtaining current release:
+..
+
+:bdg-link-primary:`Current Release <https://github.com/kokkos/kokkos/archive/refs/heads/master.zip>`
+
+
+::
+
+  unzip kokkos-master.zip
+  cd kokkos-master
+  mkdir build
+  cd build
+
+
+----------------------------------------------
+Basic Configuration Examples for Source Builds
+----------------------------------------------
+
+  You can create small shell scripts to manage configuration details, following the GPU microarchitecture-appropriate examples below.  Upon successful configuration, ``make -j`` to build, and ``make install`` to install.
+
+Serial
+~~~~~~
+::
+
+  export INSTALL_DIR = ${MY_INSTALL_DIR}
+
+  cmake \
+    -DKokkos_ARCH_NATIVE=ON \
+    -DKokkos_ENABLE_OPENMP=OFF \
+    -DKokkos_ENABLE_SERIAL=ON \
+    -DCMAKE_CXX_STANDARD=17 \
+    -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+
+OpenMP
+~~~~~~
+::
+
+  export INSTALL_DIR = ${MY_INSTALL_DIR}
+
+  cmake \
+   -DKokkos_ARCH_NATIVE=ON \
+   -DKokkos_ENABLE_OPENMP=ON \
+   -DKokkos_ENABLE_SERIAL=ON \
+   -DCMAKE_CXX_STANDARD=17 \
+   -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR \
+   -DCMAKE_BUILD_TYPE=Release ..
+
+
+CUDA
+~~~~
+
+::
+
+  export INSTALL_DIR = ${MY_INSTALL_DIR}
+
+  cmake \
+    -DKokkos_ARCH_NATIVE=ON \
+    -DKokkos_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE=OFF \
+    -DKokkos_ENABLE_CUDA=ON \
+    -DKokkos_ENABLE_SERIAL=ON \
+    -DCMAKE_CXX_STANDARD=17 \
+    -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+.. note::
+
+  VEGA90A is for AMD MI200
+  AMD_GFX942 is for AMD MI300
+
+HIP
+~~~
+::
+
+  cmake \
+    -DCRAYPE_LINK_TYPE=dynamic \
```

Comment:
That does not seem like the canonical HIP configuration.  If you want to give configuration for some LCF system then say what system it is for.